lib/ui/locations/entries/mvt_clone.mjs

```diff
@@ -94,7 +94,12 @@ export default entry => {
     }
     
     // Add the clone layer to the mapview.Map.
-    entry.mapview.Map.addLayer(entry.L)
+    try {
+      entry.mapview.Map.addLayer(entry.L)
+
+    } catch {
+      // Will catch assertation error when layer is already added.
```

Comment:
Should we not log the error still?

Reply:
Not required since it is not an error on which you can act. We could do a hasLayer check instead before addLayer but that would be more code and no performance gains.